### ISSUE TITLE
Support for IPP or other printer monitor solution

### ISSUE THREAD

**Issue by angelwzr** [louislam/uptime-kuma#2790]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

New Monitor

### 🔖 Feature description

I would really like to be able to monitor (and therefore get a notification if something goes wrong) a bunch of printers I have.
The important data is availability (current status) and ink level. 
Right now I'm using a Home Assistant IPP integration for a simple ink level display inside my dashboard but if Uptime Kuma had this protocol supported, I would switch to it in an instant.

### ✔️ Solution

Create an IPP monitor

### ❓ Alternatives

SNMP maybe?

### 📝 Additional Context

For most of the printers, afaik, it foes like this: ipp://%printer ip%/ipp/print.

**Comment by Computroniks** [comment]
I think that this could be a bit out of scope for Uptime Kuma. The application is about establishing if something is up or down, not getting detailed information about a device. For something like this, Prometheus or some other monitoring system would be better. I think that the ping check could be used for checking if the printer is online or not.

**Comment by Gunsmithy** [comment]
I don't believe they meant they want the ink levels, just that querying something like that over IPP is how they currently check uptime of a printer.
I've been using an HTTP monitor for our printer that goes down regularly but it has been insufficient. It seems like the web interface will still be up but the LPT port or whatever is down.
I've not tried a ping test on any printing ports, so not sure how accurate those are currently.
